### PR TITLE
Redesign user inactive page

### PR DESCRIPTION
**Pull request by ruanres**
**Feature/Bug description:** Redesign the pages for search institution, request form and the page to show the message after the request is sent.

**SEARCH PAGE**

**Smartphones**
search-01

**Ipad**
ipad-search

**Ipad pro**
ipad-pro-search

**FORM PAGE**

**Smartphones**
form-01

**Ipad**
ipad-form

**Ipad pro**
ipad-pro-form

**MESSAGE PAGE**

**Smartphones**
finish

**Ipad Ipad pro**
ipads-finish

**Solution:** ...

**TODO/FIXME:** Fix tests

### CODE REVIEW

File: frontend/institution/institution.js [hunk 1]
```diff
@@ -59,10 +59,16 @@ Institution.prototype.addChildInst = function addChildInst(institution){
     childIsNew ? this.children_institutions.push(institution) : null;
 };
 
+Institution.prototype.getSimpleAddress = function () {
+    if(this.address) {
```
Comment (raonismaneoto) [inline frontend/institution/institution.js]: What if there is no address? Remove the if statement or add other return to the else case

File: frontend/user/userInactive/userInactive.controller.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+(function() {
+    var app = angular.module("app");
```
Comment (raonismaneoto) [inline frontend/user/userInactive/userInactive.controller.js]: const

File: frontend/user/userInactive/userInactive.controller.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    app.controller("UserInactiveController", function UserInactiveController(AuthService, $state) {
+        var userInactiveCtrl = this;
```
Comment (raonismaneoto) [inline frontend/user/userInactive/userInactive.controller.js]: const

File: frontend/user/userRequestForm/userRequestForm.controller.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+        userReqFormCtrl.getMessage = function () {
+            const formMsg = "Para finalizar o pedido de convite, preencha suas informações institucionais";
+            const requestSentMsg = `Sua solicitação de convite foi enviada e esta em analise pelo administrador
+             de sua instituição na plataforma e-CIS. Voce recebera a confirmação em seu e-mail.`
+            return userReqFormCtrl.isRequestSent ? requestSentMsg : formMsg;
+        };
+
+        userReqFormCtrl.sendRequest = function () {
+            var dataInvite = {
```
Comment (raonismaneoto) [inline frontend/user/userRequestForm/userRequestForm.controller.js]: const

File: frontend/test/specs/user/userInactiveControllerSpec.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-(describe('Test UserInactiveController', function() {
+(xdescribe('Test UserInactiveController', function() {
```
Comment (tiagolimpe) [inline frontend/test/specs/user/userInactiveControllerSpec.js]: xdescribe?

Comment (tiagolimpe) [inline frontend/test/specs/user/userInactiveControllerSpec.js]: xdescribe?

File: frontend/user/userRequestForm/userRequestForm.controller.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+                admin_key : userReqFormCtrl.selectedInst.admin.key,
+                is_request : true,
+                type_of_invite : 'REQUEST_USER',
+                sender_name : userReqFormCtrl.user.name,
+                office : userReqFormCtrl.request.office,
+                institutional_email : userReqFormCtrl.request.email
+            };
+
+            var request = new Invite(dataInvite);
+            RequestInvitationService.sendRequest(request, userReqFormCtrl.selectedInst.key)
```
Comment (JuliePessoa) [inline frontend/user/userRequestForm/userRequestForm.controller.js]: return

File: frontend/user/userRequestForm/userRequestForm.controller.js [hunk 7]
```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+(function() {
+    angular.module('app')
+    .controller('UserRequestFormController', function (AuthService, RequestInvitationService, MessageService, $state) {
```
Comment (JuliePessoa) [inline frontend/user/userRequestForm/userRequestForm.controller.js]: js doc

File: frontend/styles/custom.css [hunk 8]
```diff
@@ -5,6 +5,12 @@
     height: 100vh;
 }
 
+@media screen and (max-width: 1366px) {
```
[the remark this answers is not in the document]
Comment (ruanres) [reply frontend/styles/custom.css]: It is the width of the iPad pro on landscape orientation

File: frontend/user/userRequestForm/userRequestForm.controller.js (hunk 6 above)
Comment (ruanres) [reply frontend/user/userRequestForm/userRequestForm.controller.js]: what should i return on this function?

Comment (JuliePessoa) [reply frontend/user/userRequestForm/userRequestForm.controller.js]: best practices and easier to test

File: frontend/user/userRequestForm/userRequestForm.controller.js [hunk 9]
```diff
@@ -0,0 +1,116 @@
+"use strict";
+
+(function() {
+		angular
+		.module('app')
+    .controller('UserRequestFormController', function (AuthService, RequestInvitationService, MessageService, $state) {
```
Comment (JuliePessoa) [inline frontend/user/userRequestForm/userRequestForm.controller.js]: does it have test?

Comment (ruanres) [reply frontend/user/userRequestForm/userRequestForm.controller.js]: not yet

File: frontend/user/searchInstitution/searchInstitution.component.js [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+                    searchInstCtrl.instNotFound = institutions.length === 0;
+                    searchInstCtrl.onSearch(institutions);
+                });
+        };
+
+        /**
+         * Select an institution and load its data
+         */
+        searchInstCtrl.select = function (institution){
+            InstitutionService.getInstitution(institution.id)
```
Comment (raonismaneoto) [inline frontend/user/searchInstitution/searchInstitution.component.js]: ToDo: Modify the inst search document to store the inst's address. Thus, this get will be unnecessary.

File: frontend/user/userInactive/userInactive.controller.js [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+    .module("app")
+    .controller("UserInactiveController", function UserInactiveController(AuthService, $state) {
+        const userInactiveCtrl = this;
+
+        userInactiveCtrl.selectedInst = {};
+
+		/**
+		 * Get the message to be showed 
+		 */
+        userInactiveCtrl.getMessage = function () {
```
Comment (raonismaneoto) [inline frontend/user/userInactive/userInactive.controller.js]: Couldn't it be in the html?

File: frontend/user/userInactive/userInactive.controller.js [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+(function() {
+    angular
+    .module("app")
+    .controller("UserInactiveController", function UserInactiveController(AuthService, $state) {
+        const userInactiveCtrl = this;
+
+        userInactiveCtrl.selectedInst = {};
+
+		/**
```
Comment (raonismaneoto) [inline frontend/user/userInactive/userInactive.controller.js]: fix all indentations

File: frontend/user/userRequestForm/userRequestForm.controller.js [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+        userReqFormCtrl.getBtnTitle = function () {
+            return userReqFormCtrl.isRequestSent ? "Início" : "Finalizar";
+        };
+
+        /**
+         * Return to the user inactive state
+         */
+        userReqFormCtrl.goBack = function () {
+            $state.go('user_inactive');
+        }
```
Comment (raonismaneoto) [inline frontend/user/userRequestForm/userRequestForm.controller.js]: missing semicolon

File: frontend/user/userRequestForm/userRequestForm.controller.js [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+         * to this institution has already been sent, otherwise
+         * it sends the request
+         */
+        function verifyAndSendRequest () {
+            if (wasInstRequested()) {
+                MessageService.showToast("Você já solicitou para fazer parte dessa instituição.");
+            } else {
+                userReqFormCtrl.sendRequest();
+            }
+        };
```
Comment (raonismaneoto) [inline frontend/user/userRequestForm/userRequestForm.controller.js]: how did it work? This semicolon couldn't be there

**Pull request merged**